rest/admin_api_test.go

```diff
@@ -314,7 +313,7 @@ function(doc, oldDoc) {
 
 			// Timeout allows us to read continuous changes after processing is complete.  Needs to be long enough to
 			// ensure it doesn't terminate before the first change is sent.
-			changesResponse := rt.Send(requestByUser("GET", fmt.Sprintf("/db/_changes?feed=continuous&since=%d&timeout=5000", since), "", "bernard"))
+			changesResponse := rt.Send(requestByUser("GET", fmt.Sprintf("/db/_changes?feed=continuous&since=%d&timeout=2000", since), "", "bernard"))
```

Comment:
Does this need to be since=%s now?

Reply:
Yikes! That was sending `2017-11-09T15:00:35.682-08:00 HTTP:  #005: GET http://localhost/db/_changes?feed=continuous&since=%!d(string=)&timeout=2000  (as bernard)`, which I'm assuming gets ignored by sync gateway .. since the test still passes with that junk since value, but probably would have failed if it had to retry.